docs/site/migration/models/operation-hooks.md

```diff
@@ -6,8 +6,38 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/migration-models-operation-hooks.html
 ---
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3952](https://github.com/strongloop/loopback-next/issues/3952)
-" %}
+Operation hooks are not supported in LoopBack 4 yet. See the
+[Operation hooks for models/repositories spike](https://github.com/strongloop/loopback-next/issues/1919)
+to follow the progress made on this subject.
+
+In the meantime, we are providing a temporary API for enabling Operation hooks
+in LoopBack 4: override `DefaultCrudRepository`'s `definePersistedModel` method
+in the model's repository.
+
+The `definePersistedModel` method of `DefaultCrudRepository` returns a model
+instance on which you can apply the
+[LoopBack 3 operation hooks](https://loopback.io/doc/en/lb3/Operation-hooks.html).
+Make sure to return the model instance from the derived class.
+
+Here is an example of a repository implementing `definePersistedModel` and
+applying an operation hook on a model instance:
```

Comment:
nitpick: I remember the `observe` method is called by a model class, not instance, not sure is if it's still the case in the legacy juggler bridge.

Reply:
Thanks! Corrected.